### PR TITLE
WIP Update style following the UWP guidelines

### PR DESCRIPTION
**Pull request by E1matheus**
Signed-off-by: Esteban Matheus <e1matheus@hotmail.com>

### Changes description

Made some changes to the style of the "Contact" page, I used winstrap assets while following the UWP guidelines:

- Changed the font size of the paragraphs.
- Reduced the number of characters per line (no more than 60)
  
contact1

- Added scale changes on every icon (on hover) to make the page more dynamic
  
contact2

- Changed the background color of the footer and added a top border to make stand out from the main 
  section.
contact3

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -6,9 +6,9 @@
   <div class="container page-not-found">
     <h1>404</h1>
 
-    <h2>Oops, sorry we can't find that page!</h2>
+    <h2>Oops, sorry Esteban we can't find that page!</h2>
```
Comment (ajsb85) [inline 404.html]: Sorry @E1matheus? rssrsrrs 🤣

File: _config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 title: glpi plugin
 email: ml-flyvemdm@teclib.com
 twitter: flyvemdm
-baseurl: "/glpi-plugin"
-url: "http://flyve.org"   
+baseurl: "/glpi-plugin-fork"
```
Comment (ajsb85) [inline _config.yml]: Hi, @E1matheus 
Remove the suffix `-fork`

File: _sass/_index.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#prueba{
+
+        background-color: blue;
+
+}
+
+/*SOMBRA PARA DAR EFECTO DE PROFUNDIDAD*/
+#boxshadow{
+
+           box-shadow: 0px 0px 5px grey;
```
Comment (ajsb85) [inline _sass/_index.scss]: indenting use 2 spaces please

File: index.html [hunk 4]
```diff
@@ -4,24 +4,24 @@
...
 <div class="container">
     <div class="col-md-12 col-md-push-12 text-center">
-      <img class="img-responsive visible-xs-inline-block visible-sm-inline-block visible-md-inline-block visible-lg-inline-block visible-xl-inline-block"
+      <img id="boxshadow" class="img-responsive visible-xs-inline-block visible-sm-inline-block visible-md-inline-block visible-lg-inline-block visible-xl-inline-block"
         alt="Flyve" src="images/logo2.png" data-source-index="2">
     </div>
     <div class="col-md-12 col-md-pull-12">
-      <div class="type-h2">
-        What is it?
+      <div class="type-h2" id="">
```
Comment (ajsb85) [inline index.html]: Unnecessary attribute `id`.

File: index.html [hunk 5]
```diff
@@ -52,20 +55,30 @@ <h3>Mobile management</h3>
...
-      </p>
+      <div class="col-md-12 col-md-push-12 text-center">
+        <img class="img-responsive icon visible-xs-inline-block visible-sm-inline-block visible-md-inline-block visible-lg-inline-block visible-xl-inline-block"
+          alt="Flyve" src="images/easy-icon.svg" data-source-index="2">
+      </div>
+
+      <div class="col-md-12 col-md-pull-12">
+        <h3><b>Easy and simple</b></h3>
+        <p class="type-p2">
+          Flyve MDM counts with an intuitive and smooth interface which <br/>
```
Comment (ajsb85) [inline index.html]: Add `<br>` is bad praxis.
May you should add padding or margin to the Paragraph element.

Comment (E1matheus) [reply index.html]: No problem, I'll work on it.

**Comment by ajsb85** [comment]
Hi, @Naylin15 
Could you help @E1matheus to resolve the conflicting files?

Hi, @E1matheus 
Could you show the responsive layout of the paragraph?

Thank you.

**Comment by E1matheus** [comment]
As requested by @ajsb85 I'm attaching some screen samples to show the responsive layout, so far my main goal has been to create a paragraph width that looks good, but that also adheres to the character limit stablished by the UWP guidelines (no more than 60 characters per line). I'm still making adjustments so the values might change.
screen 1 - desktop view
screen 2 - galaxy s5 view
screen 3 - ipad view
screen 4 - ipad view
screen 5 - glasaxy s5 view
screen 6 - ipad pro view
screen 7 - ipad pro view

**Comment by E1matheus** [comment]
Today I made a couple of new icons for the index page (mobile managemente and security sections), here are some sample images:
screen 8 - new mobile icon - iphone x view
screen 9 - new security icon - desktop view

**Comment by E1matheus** [comment]
Modified the "news" page and the post  "2017-11-30-flyve-mdm-plugin-for-glpi.md", applied some html tags in order to use the new classes for the paragraphs; kept the images and titles in markdown.
pantalla1
pantalla2-galaxys5 view
pantalla3
pantalla4-ipad view

**Pull request merged**